### ISSUE TITLE
AssertionError because of missing ngram file

### ISSUE THREAD

**Issue by BigBerny** [dimazest/google-ngram-downloader#16]
In _get_indices()_ line 190 there is a hack because 'qk' is missing in English 5gram. Unfortunately this hack is not language dependent. While 'qk' is available in other languages ([Example](http://storage.googleapis.com/books/ngrams/books/googlebooks-ger-all-5gram-20120701-qk.gz)) other indices are not ([Example](http://storage.googleapis.com/books/ngrams/books/googlebooks-ger-all-3gram-20120701-qy.gz)). That's why I get a AssertionError in line 126 when trying to access 'qy' 3grams in German. 

So instead of this hack it would be better to check if each url is valid in _iter_google_store()_. For example by replacing line 126 by:
`if request.status_code != 200:
            continue`

Thanks for your work by the way! :)
Big_Berny

**Comment by dimazest** [comment]
Thanks for the report, I'll have a look into it.

**Comment by BigBerny** [comment]
Great. I just saw it's related to https://github.com/dimazest/google-ngram-downloader/issues/9.

**Comment by dimazest** [comment]
Yes, as you can see, I don't really have time :(. Pull requests are welcome. I would just make sure that in case of an error, a warning is shown on the screen.

**Comment by tianhuil** [comment]
HI, there is a PR that solves this from [my fork](https://github.com/tianhuil/google-ngram-downloader) pending but you can pip install it in the meantime
``` bash
> pip install git+git://github.com/tianhuil/google-ngram-downloader.git@master
```
